guides/source/action_cable_overview.md

```diff
@@ -783,6 +783,10 @@ This may change in the future. [#27214](https://github.com/rails/rails/issues/27
 
 NOTE: PostgreSQL has a [8000 bytes limit](https://www.postgresql.org/docs/current/sql-notify.html) on `NOTIFY` (the command used under the hood for sending notifications) which might be a constraint when dealing with large payloads.
 
+##### Solid Cable Adapter
```

Comment:
Given that it is the default, why not put it above redis/pg?

Reply:
Make sense. Fixed.